playbooks/roles/shadowsocks/defaults/main.yml

```diff
@@ -4,3 +4,6 @@ shadowsocks_local_port: "1080"
 shadowsocks_timeout: "600"
 shadowsocks_encryption_method: "chacha20-ietf-poly1305"
 shadowsocks_tcp_fast_open: "true"
+shadowsocks_obfs_plugin: "obfs-server"
+shadowsocks_obfs_plugin_opts: "obfs=http"
+shadowsocks_obfs_client_plugin_opts: "obfs=http;obfs-host=www.xxx.com"
```

Comment:
@cpu Setting this variable on client side, obfuscates its shadowsocks traffic as the host name mentioned here. I used `x` here as in mathematics for a variable hostname, however if it is confusing then I can change it to a more suitable value. I have added this variable `shadowsocks_obfs_client_plugin_opts` to be used in the instructions for Linux based clients esp. SS-libev clients. I haven't checked it with the mirrored SS client for linux by streisand, however  I am on it.